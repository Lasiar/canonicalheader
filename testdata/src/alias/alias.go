package alias

import "net/http"

type myHeader = http.Header

func _() {
	myHeader{}.Get("TT") // want `use "Tt" instead of "TT"`
}
