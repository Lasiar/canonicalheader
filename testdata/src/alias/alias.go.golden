package alias

import "net/http"

type myHeader = http.Header

func _() {
	myHeader{}.Get("Tt") // want `use "Tt" instead of "TT"`
}
